packages/nimble-components/src/rich-text/editor/index.ts

```diff
@@ -350,6 +386,12 @@ export class RichTextEditor extends FoundationElement implements ErrorPattern {
             // Lists do not have any default paste rules, they have only input rules, so disabled paste rules
             // https://tiptap.dev/api/editor#enable-paste-rules
             enablePasteRules: false,
+            editorProps: {
+                // Validating whether the links in the pasted content belongs to the supported scheme (HTTPS/HTTP),
+                // and rendering it as a link in the editor. If not, rendering it as a plain text.
+                // Also, updating the link text content with its href as we support only the absolute link.
+                transformPasted
```

Comment:
We should be able to remove the entire transformPasted when full hyperlink support is added right? Why was that removed?

Reply:
Yesterday, we thought it was needed for validating even for hyperlinks, but seems this is an issue with Tiptap. So I have added it back and captured the information in the hyperlink user story.